src/containers/find-course/courses-filter-bar/CoursesFilterBar.tsx

```diff
@@ -1,29 +1,26 @@
+import { FC } from 'react'
 import { useTranslation } from 'react-i18next'
 import { Box } from '@mui/material'
 
 import AppSelect from '~/components/app-select/AppSelect'
 import { sortTranslationKeys } from '~/containers/find-course/courses-filter-block/CoursesFilterBlock.constants'
 import { styles } from '~/containers/find-course/courses-filter-bar/CoursesFilterBar.styles'
+import { SortProps } from '~/types'
 
-const CoursesFilterBar = () => {
+const CoursesFilterBar: FC<SortProps> = ({ setSort, sort }) => {
   const { t } = useTranslation()
 
   const handleSortBy = (value: string) => {
-    console.log(value)
+    setSort(value)
   }
 
-  const sortOptions = sortTranslationKeys.map(({ title, value }) => ({
```

Comment:
do you need to remove this map, because here you translate title

Reply:
yeah no need to translate it here , it will translate in AppSelect